docker/girder.Dockerfile

```diff
@@ -18,7 +18,13 @@ ENV GIRDER_ADMIN_PASSWORD viame
 ENV CELERY_BROKER_URL amqp://guest:guest@rabbit/
 ENV BROKER_CONNECTION_TIMEOUT 2
 
-COPY --from=builder /app/dist/ /usr/share/girder/static/viame/
+# Initialize python virtual environment
+RUN apt-get update && apt-get install -y python3.7-venv
+ENV VIRTUAL_ENV=/opt/venv
+RUN python3.7 -m venv $VIRTUAL_ENV
+ENV PATH="$VIRTUAL_ENV/bin:$PATH"
```

Comment:
This all makes me very uncomfortable.   I understand if it's needed to get the project back on its feet.

I'd prefer to:

1. Wrap this whole thing in caution tape with a big flashy comment that says this is a temp fix
2. Add a reference to an open girder issue so we can have some breadcrumb for a future developer to know what happened here.

Reply:
My guess as to why this may the problem is that there are pip installed dependencies from the base `girder/girder` image, and creating a virtual environment would allow us to install what we need to without running into a version conflict. Specifically, it was suggested that the `dogpile.cache` package contained a dependency which was leading to the conflict.

I understand that if the builds seem to be working without this change, it's tempting to just remove it, but that's the position we were in before the build errors started randomly cropping up, and we were none the wiser. I think it's worth it to leave this fix in here in case it does start occurring again, but I agree caution tape is probably a good thing.